src/main/java/core/basesyntax/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.writer;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String filePath) {
+        try (BufferedWriter bufferedWriter =
+                    new BufferedWriter(
+                            new java.io.FileWriter(
+                                    new File(filePath)
+                            )
+                    )
+        ) {
+            bufferedWriter.write(resultingReport);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
Make sure that the FileWriterImpl class is placed in the correct package. According to the checklist, service implementations should be in the `service.impl` package.